scripts/in_container/check_environment.sh

```diff
@@ -61,10 +61,16 @@ function check_db_backend {
     elif [[ ${BACKEND} == "sqlite" ]]; then
         return
     elif [[ ${BACKEND} == "none" ]]; then
-        echo "${COLOR_YELLOW}WARNING: Using no database backend!${COLOR_RESET}"
-        return
+        echo "${COLOR_YELLOW}WARNING: Using no database backend${COLOR_RESET}"
+
+        if [[ ${START_AIRFLOW} == "true" ]]; then
```

Comment:
```suggestion
        if [[ ${START_AIRFLOW=} == "true" ]]; then
```

Otherwise it will fail if START_AIRFLOW is not set at all.

Reply:
@potiuk Oh sorry, next time, I do not reproduce mistake like that. Thank you

Reply:
No... No worries :). It's a bit dark-magic knowledge of Bash and the fact that we set the options to fail when variable is not declared. So you have all the right to **not** know that :)

Reply:
ah ha Thank you!, next time I will post with test result output